### PR TITLE
Use an instance of Fuseki instead of RDFLib to service queries

### PR DESCRIPTION
**Pull request by shawa**
Opening for CI

The `dinto` object was SLOW to start and really just SAD and is now gone. `queries` is now `dinto` and the functions that returned `preparedQuery` objects now just go and run the thing.

The only API changes this introduces is that `ping` now also checks that Fuseki is available, and returns a 503 if it can't connect and run a dummy query.

The address and port are configureable with `ASCLEPIUS_FUSEKI_ADDRESS`.

First-time requests all now seem to be < 1 second.

### CODE REVIEW

File: asclepius/asclepius/main.py [hunk 1]
```diff
@@ -32,20 +34,17 @@ def handle_invalid_usage(error):
...
-def to_labelled(query_result):
-    return [{'uri': uri, 'label': label} for (uri, label) in query_result]
-
-
 @app.route("/all_drugs", methods=['GET'])
 def drugs():
     """Return a list of all drugs listed in DINTO"""
-    return jsonify(to_labelled(dinto.all_drugs()))
+    resp = dinto.all_drugs()
+    return jsonify(dinto.all_drugs())
```
Comment (houli) [inline asclepius/asclepius/main.py]: Calling `dinto.all_drugs()` twice here

File: asclepius/asclepius/main.py [hunk 2]
```diff
@@ -69,12 +68,23 @@ def ddis():
...
 
 @app.route("/ping")
 def ping():
-    return '', 204
+    DUMMY_REQUEST = 'SELECT * WHERE {?s ?p ?o} LIMIT 0'
+    DUMMY_RESPONSE = {"head": {"vars": ["s", "p", "o"]},
+                      "results": {"bindings": []}}
+    try:
+        resp = requests.post(dinto.SPARQL_ENDPOINT, data={'query': DUMMY_REQUEST})
+    except requests.exceptions.ConnectionError:
```
Comment (houli) [inline asclepius/asclepius/main.py]: `RequestException` is the superclass of all the exceptions that the requests library will throw so probably better to catch that

**Review by houli — CHANGES_REQUESTED** [review]
Need to update your requirements.txt for the tests to work on CI

**Comment by c-brenn** [comment]
updated the requirements.txt

File: asclepius/asclepius/main.py (hunk 2 above)
Comment (c-brenn) [reply asclepius/asclepius/main.py]: do we want to keep the `ping` code around seeing as there's no long boot time anymore?

Comment (22a) [reply asclepius/asclepius/main.py]: I vote to remove all status checking

Comment (houli) [reply asclepius/asclepius/main.py]: We can get rid of it I think

File: asclepius/asclepius/main.py (hunk 1 above)
Comment (c-brenn) [reply asclepius/asclepius/main.py]: removed duplicate call

File: asclepius/asclepius/main.py (hunk 2 above)
Comment (c-brenn) [reply asclepius/asclepius/main.py]: got rid of it all. much less cruft now

**Comment by c-brenn** [comment]
I think this is ready to go

**Review by houli — APPROVED** [review]

**Comment by houli** [comment]
Grand, will change the ASCLEPIUS_ADDRESS stuff to use the config thing in my PR when it's in

File: asclepius/asclepius/main.py (hunk 2 above)
Comment (shawa) [reply asclepius/asclepius/main.py]: No need for it now, yeah. Let's dump it.

**Pull request merged**